src/openrct2/Date.h

```diff
@@ -44,3 +69,28 @@ namespace OpenRCT2
         static int32_t GetDaysInMonth(int32_t month);
     };
 } // namespace OpenRCT2
+
+struct RealWorldDate
+{
+    uint8_t day;
+    uint8_t month;
+    int16_t year;
+    uint8_t day_of_week;
+};
+
+struct RealWorldTime
+{
+    uint8_t second;
+    uint8_t minute;
+    uint8_t hour;
+};
+
+extern const int16_t days_in_month[MONTH_COUNT];
+extern OpenRCT2::Date gDate;
+extern RealWorldTime gRealTimeOfDay;
+
+int32_t DateGetMonth(int32_t months);
+int32_t DateGetYear(int32_t months);
+int32_t DateGetTotalMonths(int32_t month, int32_t year);
+void DateReset();
```

Comment:
This function might be better in the Date object.

Reply:
Since the function also modifies `gRealTimeOfDay`, I moved the function to GameState instead.